rhapsody.js

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) 2016 Rhapsody International
+// Code released under the MIT license.
+// See README for more detail.
```

Comment:
Let's give them the explicit URL for the readme.
https://github.com/Rhapsody/rhapsody.js#the-mit-license-mit
This way even if rhapsody.js is copied to someone else's repo, they can still find the original license.